.github/workflows/build-android.yml

```
@@ -57,6 +53,7 @@ jobs:
           repository: 'breez/breez-sdk'
           ssh-key: ${{secrets.REPO_SSH_KEY}}
           path: 'breez-sdk'
+          ref: ${{ inputs.ref }}
```

I would suggest the same here

```suggestion
          ref: ${{ inputs.ref || github.sha }}
```